secio/Cargo.toml

```diff
@@ -18,6 +18,7 @@ no-default-features = true
 [dependencies]
 bytes = "0.5"
 futures = { version = "0.3.0" }
+futures-util = { version = "0.3.0" }
```

Comment:
No need to import separately, `pin_mut!` can be obtained directly from the futures

Reply:
got it